scripts/deployment/phase2-assets/3_deploy_fToken_collateral.ts

```diff
@@ -0,0 +1,138 @@
+    oracleTimeout: oracleTimeout(chainId, '86400').toString(), // 24 hr
+    targetName: hre.ethers.utils.formatBytes32String('USD'),
+    defaultThreshold: fp('0.0125').toString(), // 1.25%
+    delayUntilDefault: bn('86400').toString(), // 24h
+    revenueHiding: revenueHiding.toString(),
+    comptroller: networkConfig[chainId].FLUX_FINANCE_COMPTROLLER,
+  })
+  await (<Asset>await ethers.getContractAt('Asset', fUsdcCollateral)).refresh()
+
+  assetCollDeployments.collateral.fUSDC = fUsdcCollateral
+  deployedCollateral.push(fUsdcCollateral.toString())
+
+  fs.writeFileSync(assetCollDeploymentFilename, JSON.stringify(assetCollDeployments, null, 2))
+
+  /********  Deploy FToken Fiat Collateral - fUSDT  **************************/
+
+  const { collateral: fUsdtCollateral } = await hre.run('deploy-ctoken-fiat-collateral', {
+    priceTimeout: priceTimeout.toString(),
+    priceFeed: networkConfig[chainId].chainlinkFeeds.USDT,
+    oracleError: fp('0.0025').toString(), // 0.25%
+    cToken: networkConfig[chainId].tokens.fUSDT,
+    maxTradeVolume: fp('1e6').toString(), // $1m,
+    oracleTimeout: oracleTimeout(chainId, '86400').toString(), // 24 hr
+    targetName: hre.ethers.utils.formatBytes32String('USD'),
+    defaultThreshold: fp('0.0125').toString(), // 1.25%
+    delayUntilDefault: bn('86400').toString(), // 24h
+    revenueHiding: revenueHiding.toString(),
+    comptroller: networkConfig[chainId].FLUX_FINANCE_COMPTROLLER,
+  })
+  await (<Asset>await ethers.getContractAt('Asset', fUsdtCollateral)).refresh()
+
+  assetCollDeployments.collateral.fUSDT = fUsdtCollateral
+  deployedCollateral.push(fUsdtCollateral.toString())
+
+  fs.writeFileSync(assetCollDeploymentFilename, JSON.stringify(assetCollDeployments, null, 2))
+
+  /********  Deploy FToken Fiat Collateral - fDAI  **************************/
+
+  const { collateral: fDaiCollateral } = await hre.run('deploy-ctoken-fiat-collateral', {
+    priceTimeout: priceTimeout.toString(),
+    priceFeed: networkConfig[chainId].chainlinkFeeds.DAI,
+    oracleError: fp('0.0025').toString(), // 0.25%
+    cToken: networkConfig[chainId].tokens.fDAI,
+    maxTradeVolume: fp('1e6').toString(), // $1m,
+    oracleTimeout: oracleTimeout(chainId, '3600').toString(), // 1 hr
+    targetName: hre.ethers.utils.formatBytes32String('USD'),
+    defaultThreshold: fp('0.0125').toString(), // 1.25%
+    delayUntilDefault: bn('86400').toString(), // 24h
+    revenueHiding: revenueHiding.toString(),
+    comptroller: networkConfig[chainId].FLUX_FINANCE_COMPTROLLER,
+  })
+  await (<Asset>await ethers.getContractAt('Asset', fDaiCollateral)).refresh()
+
+  assetCollDeployments.collateral.fDAI = fDaiCollateral
+  deployedCollateral.push(fDaiCollateral.toString())
+
+  fs.writeFileSync(assetCollDeploymentFilename, JSON.stringify(assetCollDeployments, null, 2))
+
+  //   /********  Deploy FToken Fiat Collateral - fFRAX  **************************/
+  //   // NOT LIQUID ENOUGH YET
```

Comment:
what does liquidity matter if its just a compound pool?

Reply:
The refPerTok() literally wouldn't increase in the tests, even for a large number of blocks 😄